src/core/api/ble_secure_api.cpp

```diff
@@ -96,6 +99,19 @@ otError otBleSecureGetPeerCertificateBase64(otInstance *aInstance, unsigned char
 #endif
 
 #if defined(MBEDTLS_SSL_KEEP_PEER_CERTIFICATE)
+otError otBleSecureGetPeerCertificateDer(otInstance *aInstance, unsigned char *aPeerCert, size_t *aCertLength)
+{
+    Error error;
+
+    VerifyOrExit(aPeerCert != nullptr, error = kErrorInvalidArgs);
+    VerifyOrExit(aCertLength != nullptr, error = kErrorInvalidArgs);
```

Comment:
Suggest using `AssertPointerIsNotNull()` for checking pointers in public API implementations.

Generally, in public APIs, we use asserts instead of `VerifyOrExit()` check. These asserts can be disabled [there is an OT configuration option for this] and therefore add no code-size overhead, vs. `VerifyOrExit()` checks, which can add up slowly and increase overall code size.